README.md

```diff
@@ -2,20 +2,13 @@
 
 # Portafolio
 
-first template mobile desktop portafolio for my web desings
+First Portfolio with responsive Desktop and Mobile version
 
-# In this milestone, I did the following activities:
 
- -Updated HTML file with resposive design to fit desktop version
-
- -Updated CSS file with responsive design to fit desktop version
-
- -Added extra vector images for the desktop version
-
- -Added linters file to lint my html and css file
  
  ## Live Demo 
-- [live portfolio demo](https://cvalencia1991.github.io/Portafolio/)
+ 
+- [live portfolio demo]( https://cvalencia1991.github.io/Portafolio/)
 
 
```

Comment:
- [ ] Kindly move your authors' title to be above all the authors currently you have one author above the title.

Reply:
ok I'm going to delete the other authors